Gemfile

```diff
@@ -246,15 +246,17 @@ end
 # To develop a gem locally and override its source to a checked out repo
 #   you can use this helper method in Gemfile.dev.rb e.g.
 #
-# override_gem 'manageiq-ui-classic', :path => File.expand_path("../manageiq-ui-classic", __dir__)
+# override_gem 'manageiq-ui-classic', :path => "../manageiq-ui-classic"
 #
-def override_gem(name, *args)
+def override_gem(name, args)
```

Comment:
`args` is not necessarily a hash, it can be an array contain a version number and a hash.

Reply:
👍 done, tested passing a version and a path